apps/wallet-mobile/src/components/ScrollableView/ScrollableView.tsx

```diff
@@ -1,16 +1,6 @@
 import * as React from 'react'
-import {ScrollView, ScrollViewProps} from 'react-native'
+import {ScrollView} from 'react-native'
 
 const ScrollViewContext = React.createContext<{current: ScrollView | null}>({current: null})
 
 export const useScrollView = () => React.useContext(ScrollViewContext).current
```

Comment:
dunno, I was just pruning unused stuff, will check in another PR, still on going.

Reply:
Seems to be used to get `isScrollBarShown` to add the top border of the action footers

Reply:
@banklesss it also looks that in the new Restore funnel it was used to auto scroll, but it is wrapped within a regular view instead of ScrollableView, I've marked you to check it.